autoload/build.vim

```diff
@@ -203,23 +301,34 @@ endfunction " }}}
 " be appended to shell commands.
 "
 " Example:
-"   s:prepare_cmd_for_shell('This is %NAME%')
+"   s:prepare_cmd_for_shell('This is %NAME%',
+"         \ {'fallback': v:true'})
 " Returns:
 "   "This is 'main.cpp'"
-function! s:prepare_cmd_for_shell(str) " {{{
+function! s:prepare_cmd_for_shell(str, build_system) " {{{
   let l:str = a:str
-  let l:str = substitute(l:str, '%PATH%', escape(shellescape(expand('%:p:h')), '\'), 'g')
-  let l:str = substitute(l:str, '%NAME%', escape(shellescape(expand('%:t')), '\'),   'g')
-  let l:str = substitute(l:str, '%HEAD%', escape(shellescape(expand('%:t:r')), '\'), 'g')
+  let l:str = substitute(l:str, '%PATH%',
+        \ escape(shellescape(expand('%:p:h')), '\'), 'g')
+  let l:str = substitute(l:str, '%NAME%',
+        \ escape(shellescape(expand('%:t')), '\'),   'g')
+  let l:str = substitute(l:str, '%HEAD%',
+        \ escape(shellescape(expand('%:t:r')), '\'), 'g')
```

Comment:
You don't have to wrap at 80 characters. 100 is fine.

Reply:
Ok, I'll fix that